WORKSPACE

```diff
@@ -63,3 +65,31 @@ new_git_repository(
 load("@//bazel:python_configure.bzl", "python_configure")
 
 python_configure(name = "local_config_python")
+
+http_archive(
+    name = "io_opencensus_cpp",
+    strip_prefix = "opencensus-cpp-0.3.0",
+    urls = ["https://github.com/census-instrumentation/opencensus-cpp/archive/v0.3.0.zip"],
+)
+
+# OpenCensus depends on Abseil so we have to explicitly to pull it in.
+# This is how diamond dependencies are prevented.
+git_repository(
+    name = "com_google_absl",
```

Comment:
The name shows that `abseil` is a google project, is it?

Reply:
Yes. Any suggestions for this?